conf/nginx.conf

```diff
@@ -6,6 +6,8 @@ location __PATH__/ {
 
   client_max_body_size 50m;
 
+  more_set_headers "Access-Control-Allow-Origin: *";
```

Comment:
Hmmmokay I don't know if this was carefully thought out, this may have important security implications ? Maybe that's okay, idk

Reply:
In fact, I don’t understand well nginx, but without this line I had a cors issue in the console tab. Looking on other apps I saw that line and it helped. But there’s maybe a way to restrict this?

Reply:
You were entirely right, I don't understand why it didn't work before but now with removing this line it still works 🎉